toolkit/tools/scheduler/schedulerutils/printresults.go

```diff
@@ -188,6 +188,10 @@ func PrintBuildSummary(pkgGraph *pkggraph.PkgGraph, graphMutex *sync.RWMutex, bu
 	logger.Log.Infof("Number of failed SRPMs:            %d", len(failures))
 	logger.Log.Infof("Number of blocked SRPMs:           %d", len(unbuiltSRPMs))
 	logger.Log.Infof("Number of unresolved dependencies: %d", len(unresolvedDependencies))
+
+	logger.Log.Infof("Number of RPMs conflicts: : %d", len(rpmConflicts))
```

Comment:
Nit-pick: let's be consistent with the style above:
```suggestion
	logger.Log.Infof("Number of RPMs conflicts: : %d", len(rpmConflicts))
```